pkg/solidity-utils/contracts/helpers/InputHelpers.sol

```diff
@@ -103,4 +103,19 @@ library InputHelpers {
             previous = current;
         }
     }
+
+    /// @dev Ensure an array of amounts is sorted. As above, does not validate length or uniqueness.
+    function ensureSortedAmounts(uint256[] memory amounts) internal pure {
+        uint256 previous = amounts[0];
+
+        for (uint256 i = 1; i < amounts.length; ++i) {
```

Comment:
kinda obvious, but maybe assert length > 2?

Reply:
We use it only for tests

Reply:
As ensureSortedTokens

Reply:
I added return if length < 2